app/src/main/res/drawable/tile_square_user.xml

```diff
@@ -3,16 +3,16 @@
     <item>
         <shape android:shape="rectangle">
             <size
-                android:width="256dp"
+                android:width="384dp"
                 android:height="384dp" />
```

Comment:
File should be named `tile_user.xml` (like all other square tiles) and size should be 256x256 dp